locale/en.po

```diff
@@ -2072,6 +2072,18 @@ msgstr "Art Gallery"
 msgid "LICENSES"
 msgstr "Licenses"
 
+#: ../src/general/MainMenu.tscn:439
+msgid "GAME WIKI"
+msgstr "GAME WIKI"
```

Comment:
The name doesn't conform to our naming convention. For translation keys, underscore should act as separation: `GAME_WIKI` and since the strings are all caps, a `CAPITAL` suffix should be added: `GAME_WIKI_CAPITAL`.